packages/esm-appointments-app/src/appointments/details/appointment-details.component.tsx

```diff
@@ -15,7 +14,7 @@ const AppointmentDetails: React.FC<AppointmentDetailsProps> = ({ appointment })
   const { t } = useTranslation();
   const [isEnabledQuery, setIsEnabledQuery] = useState(false);
   const { appointmentsCount, isLoading } = usePatientAppointmentHistory(appointment.patient.uuid);
-  const { patientDetails } = usePatientDetails(appointment.patient.uuid, isEnabledQuery);
+  const { patient } = usePatient(appointment.patient.uuid);
```

Comment:
I'm 100% in support for replacing this custom hook with the standard usePatient hook... it looks like the usePatientDetails isn't used anywhere else?  If so, can you delete that file entirely?